src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html

```diff
@@ -111,6 +111,13 @@
                     <i class="bi bi-clipboard"></i>
                     {{ copyReviewTextButtonText }}
                 </button>
+                <button *ngIf="review && isTreeStyleLanguage()" type="button" class="btn btn-outline-primary" tooltipPosition="top" [disabled]="aiReviewGenerationState == 'Completed'" (click)="generateAIReview($event)">
+                    <i *ngIf="aiReviewGenerationState == 'NotStarted'" class="bi bi-stars"></i>
+                    <div *ngIf="aiReviewGenerationState == 'InProgress'" class="spinner-grow spinner-grow-sm" role="status">
+                        <span class="visually-hidden">Loading...</span>
+                    </div>
+                    {{ generateAIReviewButtonText }}
+                </button>
```

Comment:
If it's a quick fix, the button should be enabled for pretty much any tree-style language except Swagger and TypeSpec.

Reply:
I have enabled for all languages using the tree style parser